### PR TITLE
Support RGB24 and RGB16 in x11capture_linux

### PR DESCRIPTION
**Pull request by edaniels**
#### Description

Wasn't able to test 16-bit but did test 24-bit on a qemu raspi3

### CODE REVIEW

**Comment by at-wat** [comment]
@edaniels do you mind applying `size_t` fix?

**Comment by edaniels** [comment]
Done

File: pkg/driver/screen/x11capture_linux.go [hunk 1]
```diff
@@ -8,6 +8,17 @@ package screen
...
 //
+// void copyRGB24(void *dst, char *src, size_t l) { // 64bit aligned copy
+//   uint64_t *d = (uint64_t*)dst;
+//   uint64_t *s = (uint64_t*)src;
+//   l /= 8;
+//   for (size_t i = 0; i < l; i ++) {
+//     *d = *s;
+//     d++;
+//     s++;
+//   }
```
Comment (at-wat) [inline pkg/driver/screen/x11capture_linux.go:19]: It would be better to use memcpy.

File: pkg/driver/screen/x11capture_linux.go [hunk 2]
```diff
@@ -38,6 +49,17 @@ package screen
...
+// void copyRGB16(void *dst, char *src, size_t l) { // 64bit aligned copy
+//   uint64_t *d = (uint64_t*)dst;
+//   uint32_t *s = (uint32_t*)src;
+//   l /= 8;
+//   for (size_t i = 0; i < l; i ++) {
+//     *d = *s;
+//     d++;
+//     s++;
+//   }
+// }
```
Comment (at-wat) [inline pkg/driver/screen/x11capture_linux.go:61]: ditto

**Comment by edaniels** [comment]
@at-wat done!

**Review by at-wat — APPROVED** [review]
LGTM!

**Pull request merged**